docker/Dockerfile

```diff
@@ -5,7 +5,7 @@
 # It is not recommended for use beyond testing scenarios.
 ##############################################################################
 
-FROM fedora:31
+FROM fedora:33
```

Comment:
Maybe instead of `:33`, which we will forget to update probably, just use `:latest`?